hudi-client/src/main/java/org/apache/hudi/metrics/Metrics.java

```diff
@@ -52,9 +52,11 @@ private Metrics(HoodieWriteConfig metricConfig) {
     Runtime.getRuntime().addShutdownHook(new Thread(() -> {
       try {
         reporter.report();
-        getReporter().close();
+        if (getReporter() != null) {
+          getReporter().close();
+        }
       } catch (Exception e) {
-        e.printStackTrace();
+        LOG.error("Error while closing reporter", e);
```

Comment:
How about changing the error level to warn?

Reply:
got it